cmd/tether/main_darwin.go

```diff
@@ -26,7 +27,12 @@ import (
 )
 
 func main() {
-	defer halt()
+	defer func() {
```

Comment:
Do `defer` instructions get run when something PANICs?  Also, logrus has a panic handler.  See [here](https://github.com/deferpanic/dp-logrus) for an example.